### PR TITLE
Fix ACPI tests

### PR DESCRIPTION
**Pull request by tweksteen**
Use regular expressions instead of strict string matching.

### CODE REVIEW

File: source/tool/tests/software/test_util.py [hunk 1]
```diff
@@ -48,12 +48,22 @@ def _chipsec_util(self, arg, helper_class=mock_helper.TestHelper):
...
         util.set_logfile(self.log_file)
         err_code = util.main(["chipsec_utils.py"] + arg.split())
         logger.logger().close()
         self.log = open(self.log_file).read()
         self.assertEqual(err_code, 0)
 
+    def _assertLogValue(self, name, value):
+        """Shortcut to validate the output.
+
+        Assert that at least one line exists wihtin the log which matches the
```
Comment (af3556) [inline source/tool/tests/software/test_util.py]: sp: s/wihtin/within/

File: source/tool/tests/software/test_util.py [hunk 2]
```diff
@@ -48,12 +48,22 @@ def _chipsec_util(self, arg, helper_class=mock_helper.TestHelper):
...
+        expression: \s*name\s*[:=]\s*value.
+        """
+        self.assertRegexpMatches(self.log,
+                                 "\s*{}\s*[:=]\s*{}".format(name, value))
```
Comment (af3556) [inline source/tool/tests/software/test_util.py]: regex ought to be firmly "anchored":
`(^|\s+){}\s*[:=]\s*{}(\s+|$)`
i.e. explicitly bound the start of the name and end of the value. e.g. to distinguish
DSDT: 0x00000000
X_DSDT: 0x0000000000000000

File: source/tool/tests/software/test_util.py [hunk 3]
```diff
@@ -48,12 +48,22 @@ def _chipsec_util(self, arg, helper_class=mock_helper.TestHelper):
...
         err_code = util.main(["chipsec_utils.py"] + arg.split())
         logger.logger().close()
         self.log = open(self.log_file).read()
         self.assertEqual(err_code, 0)
 
+    def _assertLogValue(self, name, value):
+        """Shortcut to validate the output.
+
+        Assert that at least one line exists wihtin the log which matches the
+        expression: \s*name\s*[:=]\s*value.
```
Comment (af3556) [inline source/tool/tests/software/test_util.py]: suggest just using a description of the expression here (no point in copying code into the comment)...:

expression of the form: name [:=] value

**Comment by af3556** [comment]
tweak reqd for the regex, otherwise lgtm

File: source/tool/tests/software/test_util.py (hunk 1 above)
Comment (tweksteen) [inline source/tool/tests/software/test_util.py]: Done.

File: source/tool/tests/software/test_util.py (hunk 3 above)
Comment (tweksteen) [inline source/tool/tests/software/test_util.py]: Done.

**Pull request closed without merging**